core/src/main/java/com/chopshop166/chopshoplib/maps/SwerveDriveMap.java

```diff
@@ -114,6 +120,10 @@ public SwerveDriveMap(final SwerveModule frontLeft, final SwerveModule frontRigh
         this.kinematics = new SwerveDriveKinematics(this.frontLeft.getLocation(),
                 this.frontRight.getLocation(), this.rearLeft.getLocation(),
                 this.rearRight.getLocation());
+        this.estimator = new SwerveDrivePoseEstimator(kinematics, new Rotation2d(),
```

Comment:
The default standard deviations are quite a bit different than what we've been testing with. It would probably be good to add `stateStdDevs` and `visionMeasurementStdDevs` as arguments to the constructor.